lib/ndr_dev_support/rake_ci/commit_cop/renamed_migration.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'concerns/deputisable'
+
+module NdrDevSupport
+  module RakeCI
+    module CommitCop
+      # This cop checks for renamed migrations allowing for unscoped to scoped name changes
+      class RenamedMigration
+        include Deputisable
+
+        def check(changes)
+          renamed_migrations = changes[:renamed].select(&migration_file?)
+          return if renamed_migrations.empty?
+          return if renamed_migrations.all? do |old_file, new_file|
+            unscoped_migration_file?.call(old_file) && !unscoped_migration_file?.call(new_file)
```

Comment:
The `SCHEMA_VERSIONS` table only tracks the timestamp as the migration identifier, so this check could generate false positives; it wouldn't be problematic to rename the Ruby classes, e.g. in the case of mis-spelling or a need for additional clarity down the line.